libs/ui/src/lib/tabs/tab-group.component.stories.ts

```diff
@@ -65,3 +67,73 @@ Mobile.parameters = {
     defaultViewport: 'iphone12mini',
   },
 };
+
+const TemplateWithIcon: Story<TabComponent> = (args: TabComponent) => ({
+  props: args,
+  template: `
+<cvi-ng-tab-group>
+  <cvi-ng-tab
+    title="Tab 1"
+    [iconName]="iconName"
+    [svgClass]="'svg-class'"
+    [iconPosition]="iconPosition"
+    [iconHeight]="iconHeight"
+    [gap]="gap"
+    >{{ content }}</cvi-ng-tab
+  >
+  <cvi-ng-tab title="Tab 2, with a form">
+    <div [formGroup]="form">
```

Comment:
See [this comment](https://github.com/e-gov/cvi/pull/165#discussion_r1296387418).